python/ql/test/experimental/dataflow/fieldflow/test_dict.py

```diff
@@ -45,6 +45,16 @@ def test_dict_update():
     SINK(d["key"])  # $ flow="SOURCE, l:-1 -> d['key']"
     SINK(d.get("key"))  # $ flow="SOURCE, l:-2 -> d.get(..)"
 
+
+@expects(2) # $ unresolved_call=expects(..) unresolved_call=expects(..)(..)
```

Comment:
```suggestion
```

Reply:
See https://github.com/github/semmle-code/actions/runs/8112809865/job/22174818063#step:26:5473

Reply:
thanks :+1: I forgot to run these locally